.env.common

```
@@ -0,0 +1,57 @@
+NODE_ENV=production
+HOST=0.0.0.0
+
+# === Email Service Configuration ===
+EMAIL_SERVICE=smtp
+SMTP_HOST=mail.utia.cas.cz
+SMTP_PORT=465
+SMTP_SECURE=true
+SMTP_REQUIRE_TLS=false
+SMTP_AUTH=true
+SMTP_USER=prusek@utia.cas.cz
+SMTP_PASS=M1i2c3h4a5l6
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SMTP_PASS key should go before the SMTP_PORT key

(UnorderedKey)